Tests/xcprojTests/PBXFileElementSpec.swift

```diff
@@ -36,26 +36,32 @@ final class PBXFileElementSpec: XCTestCase {
     func test_init_failsIfSourceTreeIsMissing() {
         var dictionary = testDictionary()
         dictionary.removeValue(forKey: "sourceTree")
+        let data = try! JSONSerialization.data(withJSONObject: dictionary, options: [])
+        let decoder = JSONDecoder()
         do {
-            _ = try PBXFileElement(reference: "ref", dictionary: dictionary)
```

Comment:
It's probably worth making an extension on PBXObject or Decodable which is `init(jsonDictionary: [String: Any]` that handles all this boilerplate

Reply:
Yeah good idea, I'll do it @yonaskolb

Reply:
Done @yonaskolb

Reply:
Nice! I think you could also put an extension on PBXObject as well that is an init with reference
```
init(reference: String, jsonDictionary: [String: Any]
```
That would remove a bit of the boilerplate in tests and other places